packages/shared/src/components/search/SearchResults/SearchResultsLayout.tsx

```diff
@@ -0,0 +1,151 @@
+import React, {
+  PropsWithChildren,
+  ReactElement,
+  useContext,
+  useMemo,
+} from 'react';
+import { useRouter } from 'next/router';
+import { PageWidgets } from '../../utilities';
+import { StraightArrowIcon } from '../../icons';
+import {
+  providerToIconMap,
+  providerToLabelTextMap,
+  SearchProviderButton,
+} from '../SearchPanel';
+import { SearchProviderEnum, SearchSuggestion } from '../../../graphql/search';
+import { useSearchResultsLayout } from '../../../hooks/search/useSearchResultsLayout';
+import { ActiveFeedContext } from '../../../contexts';
+import { FeedItemType } from '../../cards/common';
+import { FeedItem, PostItem } from '../../../hooks/useFeed';
+import { LogEvent, Origin, TargetType } from '../../../lib/log';
+import { useLogContext } from '../../../contexts/LogContext';
+import { webappUrl } from '../../../lib/constants';
+import { SearchResultsTags } from './SearchResultsTags';
+import { SearchResultsSources } from './SearchResultsSources';
+
+type SearchResultsLayoutProps = PropsWithChildren;
+
+const AICta = {
+  Icon: providerToIconMap[SearchProviderEnum.Chat],
+  Label: providerToLabelTextMap[SearchProviderEnum.Chat],
+};
+
+function isItemPost(item: FeedItem): item is PostItem {
+  return 'post' in item && item.type === FeedItemType.Post;
+}
+
+export const SearchResultsLayout = (
+  props: SearchResultsLayoutProps,
+): ReactElement => {
+  const { children } = props;
+
+  const { isSearchResultsUpgrade } = useSearchResultsLayout();
+  const { items = [] } = useContext(ActiveFeedContext);
+  const {
+    query: { q: query },
+    push,
+  } = useRouter();
+  const { logEvent } = useLogContext();
+  const isLoading = !items.length;
```

Comment:
Search results could have no results right, so this is not exactly correct? If you need loading specifically you can add it to `ActiveFeedContext`

Reply:
Not really needed, I'm removing! Thanks for pointing at :)